spec/javascripts/mobile/mobile_comments_spec.js

```diff
@@ -82,14 +82,8 @@ describe("Diaspora.Mobile.Comments", function(){
   describe("createComment", function () {
     beforeEach(function() {
       spec.loadFixture("aspects_index_mobile_post_with_comments");
-      var commentBoxHtml = spec.fixtureHtml("comments_mobile_commentbox");
       var link = $(".stream .comment-action").first();
       Diaspora.Mobile.Comments.showCommentBox(link);
-      jasmine.Ajax.requests.mostRecent().respondWith({
```

Comment:
It's not a test, it's enclosed in the `beforeEach()` function. This line [is buggy](https://travis-ci.org/diaspora/diaspora/jobs/150391203#L813) with jQuery 2 and since it is not relevant for the test, I thought it was not a big deal to remove it.